mender-deb-package

```diff
@@ -50,6 +50,24 @@ checkout_repo() {
   git checkout $VERSION
 }
 
+install_go() {
+ local GOLANG_VERSION=1.14.7
```

Comment:
I think default should be `1.17.6`, which is what we currently have for all.

Reply:
changed.